docker/install-dynamic-plugins.py

```diff
@@ -230,7 +231,7 @@ def main():
 
         archive = os.path.join(dynamicPluginsRoot, completed.stdout.decode('utf-8').strip())
 
-        if not package_is_local:
+        if not (package_is_local or skipIntegrityCheck):
```

Comment:
This would cause integrity checks to occur for local packages as well, which I don't think we want?

Reply:
I don't think so.

If `package_is_local` is `true`, then `package_is_local or skipIntegrityCheck` is always `true` due to the `or`.
So that in this case `not(package_is_local or skipIntegrityCheck)` is `false`.

So as soon as the package is local *`or`* we skip integrity check, we would *not* enter the conditional line.

That's how I understand it and thought it.

Reply:
ok nevermind then, I think that'll be fine then. I just still have the same argument as [above](https://github.com/janus-idp/backstage-showcase/pull/704#discussion_r1380481117) regarding the redundancy of the `skipIntegrityCheck` check though.